docs/projects/reverse-a-string/_posts/2019-10-03-lua.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Reverse a string in Lua
+layout: default
+last-modified: 2019-10-01
+featured-image:
+tags: [javascript, reverse-a-string, string-reversal]
+authors:
+  - herrfugbaum
```

Comment:
Feel free to update this to include your username. Then, [add yourself to the author file](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml).